### PR TITLE
Update to v0.21.1

### PR DESCRIPTION
**Pull request by nevrending**
Original build instructions: https://github.com/Bitcoin-ABC/bitcoin-abc/blob/master/doc/build-unix.md

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,14 @@
...
+[![ImageLayers](https://images.microbadger.com/badges/image/bitwyre/crypto-bitcoin-cash.svg)](https://microbadger.com/images/bitwyre/crypto-bitcoin-cash)
+
+Runs the Bitcoin ABC bitcoind full-node in a container.
+
+Copyright (C) 2019, Bitwyre Technologies LLC.
+
+## Authors
+
+- [Yefta Sutanto](https://github.com/nevrending)
+- [Aditya Kresna](https://github.com/ujang360)
```
Comment (nevrending) [inline README.md]: Hehe @Ujang360

**Review by dendisuhubdy — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (dendisuhubdy) [reply README.md]: 🎉

Comment (Ujang360) [reply README.md]: :joy:

**Review by Ujang360 — APPROVED** [review]
Looks good :+1:

**Pull request merged**